modules/vocabularies/src/main/frontend/src/vocabularyDirectory.jsx

```diff
@@ -17,113 +17,94 @@
 //  under the License.
 //
 
-import React from "react";
+import React, {useEffect} from "react";
 
 import { 
-  Button, 
-  Grid, 
+  Button,
+  Grid,
   LinearProgress,
-  makeStyles,
-  Typography 
+  Typography
 } from "@material-ui/core";
 
 import VocabularyTable from "./vocabularyTable";
 
 const Status = require("./statusCodes.json");
 
-const useStyles = makeStyles(theme => ({
-  root: {
-    flexGrow: 1,
-  },
-  button: {
-    margin: theme.spacing(1),
-    textTransform: "none",
-    color: "white",
-    borderRadius: 3,
-    border: 0
-  }
-}));
-
 /*
-  This function reformats the data from the local source into the format of data from the remote source. 
-  This allows for the reusing of the same code to display a table for both the local and remote vocabularies.
+  This function reformats the vocab data for more light unified representation.
 */
-function reformat(data) {
-  data.map((vocabulary) =>  {
-      vocabulary.ontology = {
-        acronym: vocabulary["identifier"],
-        name: vocabulary["name"]
-      };
-      vocabulary.released = vocabulary["jcr:created"];
-  });
-  return data;
+function reformat(data, type) {
+  let vocabs = [];
+  data.map(vocab =>  vocabs.push({
+          status: vocab.status,
+          acronym: type == "remote" ? vocab.ontology.acronym : vocab.identifier,
+          name: type == "remote" ? vocab.ontology.name : vocab.name,
+          source: vocab.source,
+          description: vocab.description,
+          released: type == "remote" ? vocab.released : vocab["jcr:created"],
```

Comment:
I wonder if we should do this instead:

```suggestion
          released: type == vocab.released || vocab["jcr:created"],
```

This way it always uses `vocab.released` if available, and always falls back to `jcr:created` if not. WDYT?